docs/samples/v1beta1/torchserve/README.md

```diff
@@ -39,11 +39,9 @@ torch-model-archiver --model-name mnist --version 1.0 \
 - For PVC user please refer to [model archive file generation](./model-archiver/README.md) for auto generation of MAR files from
 the model and dependent files.
 
-- The [config.properties](./v1/config.properties) file includes the flag `service_envelope=kfserving` to enable the KServe inference protocol.
```

Comment:
Can you add instruction here that `service_envelope` is deprecated and use `enable_envvars_config` instead.